README.md

```diff
@@ -11,22 +11,22 @@ download links are also provided below.
 
 ### Gradle
 ```groovy
-compile "com.smartcar.sdk:java-sdk:2.2.1"
+compile "com.smartcar.sdk:java-sdk:2.2.2"
```

Comment:
hmm, should we do a bigger version bump? `2.3.0` cause we are introducing a new feature (permissions caching) and also breaking the interface a bit by requiring developers to handle for a new exception.